pkgs/development/python-modules/django-ordered-model/default.nix

```
@@ -0,0 +1,56 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy27
+
+# buildtime
+, setuptools
+
+# runtime
+, django
+
+# tests
+, python
+, djangorestframework
+}:
+
+buildPythonPackage rec {
+  pname = "django-ordered-model";
+  version = "3.7.4";
+  format = "pyproject";
+
+  disabled = isPy27;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-8li5diUlwApTAJ6C+Li/KjqjFei0U+KB6P27/iuMs7o=";
+  };
+
+  nativeBuildInputs = [
+    setuptools
+  ];
+
+  checkInputs = [
```

```suggestion
  nativeCheckInputs = [
```
Please use `nativeCheckInputs` not `checkInputs`.
> Test dependencies, like e.g. the test runner, should be added to nativeCheckInputs.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#buildpythonpackage-function-buildpythonpackage-function